src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,34 @@
-export const PostInfo = () => (
-  <>Put the post here</>
-);
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+import './PostInfo.scss';
+
+export const PostInfo = ({ post }) => {
+  const { title, user, body, comments } = post;
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">
+          {title}
+        </h3>
+
+        <p>
+          {' Posted by  '}
+
+          <UserInfo user={user} />
+        </p>
+      </div>
+
+      <p className="PostInfo__body">
+        {body}
+      </p>
+      <hr />
+      {comments.length > 0 ? (
```

Comment:
Don't forget about formatting
```suggestion
      <p className="PostInfo__body">
        {body}
      </p>
      
      <hr />
      
      {comments.length > 0 ? (
```